pages/system/card-tabs.js

```diff
@@ -54,55 +69,67 @@ const TAB_GROUP_FOUR = [
   { value: '3', label: 'Pisces' },
   { value: '4', label: 'Aries' },
 ];`}
-        </System.CodeBlock>
-        <br />
-        <System.P>Define the CardTab value states and handle the state when a tab is changed.</System.P>
-        <br />
-        <System.CodeBlock>
-          {`state = {
-  eighteen: '2',
-  nineteen: null,
-};
-
-_handleChange = (e) => {
-  this.setState({ [e.target.name]: e.target.value });
-};`}
         </System.CodeBlock>
         <br />
         <System.P>Declare the CardTabGroup component.</System.P>
         <br />
         <System.CodeBlock>
-          {`<System.CardTabGroup
-  name="eighteen"
-  options={TAB_GROUP_TWO}
-  value={this.state.eighteen}
-  onChange={this._handleChange}
-/>
+          {`class ExampleOne extends React.Component {
+   state = { exampleOne: '1' }
+
+   _handleChange = (e) => this._handleChange(
+     { [e.target.name]: e.target.value }
+   )
+
+   render() {
+       return(
+         <CardTabGroup
+           name="exampleOne"
+           options={TAB_GROUP_TWO}
+           value={this.state.exampleOne}
+           onChange={this._handleChange}
+         />
+       )
+   }
+}
+
+class ExampleTwo extends React.Component {
+   state = { exampleTwo: '4' }
+
+   _handleChange = (e) => this._handleChange(
```

Comment:
This is not correct, you need

`_handleChange = e => this.setState({ [e.target.name]: e.target.value });`